jobs/mysql/templates/my.cnf.erb

```diff
@@ -45,7 +45,7 @@ nice      = 0
 wsrep_provider=/var/vcap/packages/mariadb/lib/plugin/libgalera_smm.so
 wsrep_provider_options="gcache.size=<%= p('cf_mysql.mysql.gcache_size') %>M;pc.recovery=TRUE"
 wsrep_cluster_address="gcomm://<%= cluster_ips.join(",") %>"
-wsrep_node_address='<%= network_ip %>:<%= p('cf_mysql.mysql.galera_port') %>'
+wsrep_node_address='<%= p('cf_mysql.mysql.advertise_host') || network_ip %>:<%= p('cf_mysql.mysql.galera_port') %>'
```

Comment:
This is the only use of `network_ip`; wouldn't it make more sense to actually change it where it's assigned?